AUTHORS

```diff
@@ -1,27 +1,59 @@
+Adam Enger <adamenger@gmail.com>
```

Comment:
No need to check this in. I'll take care of this in a separate PR near release time.